io.asgardio.java.oidc.sdk/src/main/java/io/asgardio/java/oidc/sdk/bean/AuthenticationInfo.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.asgardio.java.oidc.sdk.bean;
+
+import com.nimbusds.jwt.JWT;
+import com.nimbusds.oauth2.sdk.token.AccessToken;
+import com.nimbusds.oauth2.sdk.token.RefreshToken;
+
+import java.io.Serializable;
+
+/**
+ * A data model class to define the Authentication Info element.
+ */
+public class AuthenticationInfo implements Serializable {
+
+    private static final long serialVersionUID = 976008884476935474L;
+
+    private User user;
+    private AccessToken accessToken;
+    private RefreshToken refreshToken;
+    private JWT idToken;
+
+    /**
+     * Returns the authenticated user.
+     *
+     * @return {@link User} object for the authenticated user.
+     */
+    public User getUser() {
+
+        return user;
+    }
+
+    /**
+     * Sets the user of the authentication.
+     *
+     * @param user The user for the authentication.
+     */
+    public void setUser(User user) {
+
+        this.user = user;
+    }
+
+    /**
+     * Returns the access token.
+     *
+     * @return The {@link AccessToken}.
+     */
+    public AccessToken getAccessToken() {
```

Comment:
Better to not to expose nimbus classes from the SDK APIs.

Reply:
Fixed with https://github.com/asgardio/asgardio-java-oidc-sdk/pull/12/commits/999e40f1cc57d0a5b510d4c734790e448705b777.